assets/styles/components/_Boxes.scss

```diff
@@ -191,6 +220,128 @@
 		}
 	}
 
+	.elementor-col-50,
+	.elementor-col-33 {
+		position: relative;
+		padding: 20px;
+
+		.elementor-widget-wrap {
+			border-radius: $border-radius-16;
+			align-items: flex-start;
+		}
+
+		.elementor-widget-image.wide .elementor-image {
+			padding: 0;
+		}
+
+		.elementor-widget-image {
+			overflow: hidden;
+			border-top-left-radius: $border-radius;
+			border-top-right-radius: $border-radius;
+
+			.elementor-image {
+				display: block;
+				height: 100%;
+				padding: 30px 40px 0;
+
+				a {
+					display: block;
+					height: 100%;
+				}
+
+				img {
+					display: block;
+					margin: auto;
+					width: auto !important;
+					max-width: 100%;
+					// height: 100%;
+				}
+			}
+		}
+
+		.elementor-text-editor {
```

Comment:
elementor-widget-text-editor if possible, this is only existing class for streamlined elementor DOM we plan to deploy soon

Reply:
Changed